Core/Src/compute.c

```diff
@@ -98,10 +124,20 @@ bool compute_charger_connected()
 // 	return;
 // }
 
-void compute_set_fan_speed(uint8_t new_fan_speed)
+//? Change timers to not 1 and 8 since they are advanced timers?
+uint8_t compute_set_fan_speed(TIM_HandleTypeDef* pwmhandle, fan_select_t fan_select, uint8_t duty_cycle)
```

Comment:
note to myself, we are planning on figuring out which pwmhandle to pass in when we create compute struct